libs/utils/api/.eslintrc

```diff
@@ -1 +1,3 @@
-{ "extends": "../../../.eslintrc.json", "rules": {}, "ignorePatterns": ["!**/*"] }
+{ "extends": "../../../.eslintrc.json", "rules": {
```

Comment:
Look like some wrong prettier here

Reply:
🤷  I don't even remember [touching this file](https://github.com/island-is/island.is/commit/650c66c0f37c3513cb365f94ff5dd6750bc5f7bb#diff-dab8984f2c6d62a1a30064e49788515551d4f4b0278dce04746e4ea094259fc1), but there are no lint or formatting errors